### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Artanic30**

### CODE REVIEW

**Comment by alienmao** [comment]
The ui is better than the previous version.  👍

File: src/components/Aside.vue [hunk 1]
```diff
@@ -12,21 +16,25 @@
 export default {
   data () {
     return {
-      msg: 'Welcome to Shanghaitech online judge!Click on one of your courses to the right, or on the Account menu above.'
+      msg: 'Welcome to Shanghaitech online judge! Click on one of your courses to the right, or on the Account menu above.',
```
Comment (alienmao) [inline src/components/Aside.vue]: may be 'Shanghai Tech' is better
Click one of your courses on the right

File: src/components/Aside.vue [hunk 2]
```diff
@@ -12,21 +16,25 @@
...
+      img: require('../assets/logo.png')
     }
   }
 }
 </script>
 <style>
```
Comment (alienmao) [inline src/components/Aside.vue]: add scoped to avoid css pollution

File: src/components/Home.vue [hunk 3]
```diff
@@ -1,22 +1,24 @@
...
-      </div>
-      <div class="text item">
-        <span>{{a.address}}</span>
-      </div>
-      <div class="bottom" style="float: left">
-        <span>状态栏</span>
-      </div>
-    </el-card>
+    <el-row style="float: left;margin: 4% 0 10px 0;width: 100%">
+        <span style="font-size: 30px">Your Courses</span>
```
Comment (alienmao) [inline src/components/Home.vue]: `
<el-row class="courses-title">
<el-col>
<span>
</span>
</el-col>
</el-row>
`

write less inline css styles

File: src/components/Home.vue [hunk 4]
```diff
@@ -1,22 +1,24 @@
...
+    </el-row>
+    <el-row style="float: left;margin: 4% 0 10px 0;width: 100%">
```
Comment (alienmao) [inline src/components/Home.vue]: the same problem

File: src/components/Home.vue [hunk 5]
```diff
@@ -1,22 +1,24 @@
...
+       <span style="font-size: 20px">{{ semester }}</span>
+    </el-row>
+       <el-card class="box-card" v-for="a in TD" :key="a" style="height: 250px;width: 400px;margin: 3% 5% 5% 0">
```
Comment (alienmao) [inline src/components/Home.vue]: your can use :key="a.name"
and the variable name TD seems too difficult for others to understand what it means

Comment (alienmao) [inline src/components/Home.vue]: use 
`
.box-card {
   margin: 3% 5% 5% 0 !important;
}
`
to overwrite the index.css css property

File: src/components/Home.vue [hunk 6]
```diff
@@ -1,22 +1,24 @@
...
+         <el-button style="background-color: #fff1f1;height: 250px;width: 400px;padding: 0 0 0 0;">
+                <el-row style="margin: 20px 0 10px 4%">
+                  <span style="float: left;font-size: 40px;font-style: inherit">{{ a.name }}</span>
+                </el-row>
+                <el-row style="height: 100px;margin: 20px 0 0 4%">
+                    <span style="font-style: italic;float: left">{{ a.address }}</span>
+                </el-row>
+                <el-row style="background-color: black;height: 80px" type="flex">
+                    <span style="margin: 30px 0 0 4%;color: white">{{ a.state }}</span>
```
Comment (alienmao) [inline src/components/Home.vue]: same problem,
<el-row><el-col> should always use together to achieve a proper layout

File: src/components/Home.vue [hunk 7]
```diff
@@ -26,14 +28,17 @@ export default {
     return {
       semester: 'Fall 2018',
       TD: [{
```
Comment (alienmao) [inline src/components/Home.vue]: variable name is also difficult to understand

File: src/components/Index.vue [hunk 8]
```diff
@@ -1,19 +1,22 @@
...
-        <el-main>
-          <v-home></v-home>
-        </el-main>
-      </el-container>
-    </el-container>
+    <el-row style="background-color: #A40006">
+      <el-col :span="2" style="float: right"><v-nav></v-nav></el-col>
+    </el-row>
+    <el-row style="height: 100%">
+      <rl-col>
```
Comment (alienmao) [inline src/components/Index.vue]: what is rl-col

File: src/components/Navigation.vue [hunk 9]
```diff
@@ -10,7 +10,7 @@
             active-text-color="#ffd04b"
             router
             >           <!--  todo:箭头与文字重合及箭头颜色   -->
-            <el-submenu index="2">
+            <el-submenu index="2" style="float: right">
```
Comment (alienmao) [inline src/components/Navigation.vue]: less inline css style

File: src/components/Index.vue (hunk 8 above)
Comment (Artanic30) [reply src/components/Index.vue]: hour

File: src/components/Home.vue (hunk 5 above)
Comment (Artanic30) [reply src/components/Home.vue]: Got it

**Pull request merged**